reconcile/slack_usergroups.py

```diff
@@ -243,6 +243,8 @@ def get_desired_state(slack_map, pagerduty_map):
         workspace_name = workspace['name']
         usergroup = p['handle']
         description = p['description']
+        if usergroup == 'app-sre-escalation-undefined':
```

Comment:
I'm not a huge fan of handling specific usergroups in different ways in our code. Is there anyway this could be done in a more general way that would make it obvious that it is ignored when looking at the resource from app-interface?

Reply:
this permission seems to be the only one with an empty array for the channel field. can you leverage that?

disclaimer: I don't know too much about permissions yet. so this is more of an observation than a real suggestion

Reply:
Good point, I did a quick filter in the code, and that empty channels does seem to be only this one. How does this sound to you @steveteahan ?